### ISSUE TITLE
hpar update

### ISSUE THREAD

**Issue by fmarchildon** [lgatto/hpar#3]
Dear Laurent,

Thanks for creating hpar. I appreciate your package very much. Any chance you will be integrating the human secretome and membrane proteome to hpar soon?

https://www.proteinatlas.org/humanproteome/tissue/secretome

Kind regards,

**Comment by lgatto** [comment]
Dear @fmarchildon - thank you for your interest. 

We are indeed going to update the `hpar`. I haven't found a way to directly download the secretome, but this query seems to do the trick
```
https://www.proteinatlas.org/search/sa_location:Intracellular+and+membrane%2CSecreted+-+unknown+location%2CSecreted+in+brain%2CSecreted+in+female+reproductive+system%2CSecreted+in+male+reproductive+system%2CSecreted+in+other+tissues%2CSecreted+to+blood%2CSecreted+to+digestive+system%2CSecreted+to+extracellular+matrix
```
These data will be included as a new `secretome` data set in the package.

Cc @ManonMartin

**Comment by lgatto** [comment]
@ManonMartin - could you also add the secretome data to the latest version on github, please.

**Comment by lgatto** [comment]
The secretome data is not included in the package (versions 1.32.1 and 1.33.1).
